include/hypercall_logging.h

```diff
@@ -3,13 +3,18 @@
 #include <stdlib.h>
 #include <assert.h>
 
+#define MAGIC_VALUE 0x8507fae1 /* crc32("busybox") */
+#include "libhc/hypercall.h"
+
 typedef enum {
     HC_CMD_LOG_LINENO = 0,
-    HC_CMD_LOG_SETFILE = 1
+    HC_CMD_LOG_ENV_ARGS = 1
```

Comment:
We don't need setfile anymore?

Reply:
correct, that was a possible optimization that was being considered and is, at best, being pushed back for now.

currently, it just tells you which file it is every logging event, if this becomes a performance issue we can add this back.